client/src/components/_common/Button/Button.module.css

```diff
@@ -1,71 +1,47 @@
-.c-button {
-  composes: c-button from '../../../styles/components/c-button.css';
-}
-.c-button:focus,
-[class*='c-button--']:focus {
-  outline: none;
+.root {
+  composes: c-button from '../../../styles/components/c-button--new.css';
 }
```

Comment:
**Explanation:** In this whole file, **all** button module styles are now pulled from `c-button--new.css`.

<details><summary>Future Changes</summary>

1. [FP-546](https://jira.tacc.utexas.edu/browse/FP-546): Replace `c-button.css` content with `c-button--new.css` content.
2. (Unscheduled) Load stylesheet from [TACC/Core-Styles `c-button.css`](https://github.com/TACC/Core-Styles/blob/main/source/_imports/components/c-button.css) instead ([example](https://github.com/TACC/Core-CMS/pull/466/files#diff-80e9452c1331968133e13bc20494a63929fa1d2221c9f9bc580ead19fab33f6eR12)).

</details>